### PR TITLE
feat: service manage

### PR DESCRIPTION
**Pull request by NicolaCage**

### CODE REVIEW

File: app/assets/modules/Layout/routes.tsx [hunk 1]
```diff
@@ -7,6 +7,11 @@ const ServerMetrics = lazy(() => import('@assets/modules/ServerMetrics'));
 const VersionStatistic = lazy(() => import('@assets/modules/VersionStatistic'));
 const LeaderDistribution = lazy(() => import('@assets/modules/LeaderDistribution'));
 const PartitionDistribution = lazy(() => import('@assets/modules/PartitionDistribution'));
+const serviceInfo = lazy(() => import('@assets/modules/ServiceManage/ServiceInfo'));
```
Comment (hetao92) [inline app/assets/modules/Layout/routes.tsx:10]: serviceInfo -> ServiceInfo

File: app/assets/modules/ServiceManage/ConfigInfo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      default:
+        break;
+    }
+  }
+
+  renderTooltip=text=>{
+    return <Tooltip placement="top" title={text} > 
+      <svg  aria-hidden="true">
+        <use xlinkHref={'#iconhelp'}></use>
+      </svg>
```
Comment (hetao92) [inline app/assets/modules/ServiceManage/ConfigInfo/index.tsx:65]: replace it with Icon Component in /assets/components/Icon

File: app/assets/modules/ServiceManage/PartitionInfo/index.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+      default:
+        break;
+    }
+  }
+
+  renderTooltip=text=>{
+    return <Tooltip placement="top" title={text} > 
+      <svg  aria-hidden="true">
+        <use xlinkHref={'#iconhelp'}></use>
+      </svg>
```
Comment (hetao92) [inline app/assets/modules/ServiceManage/PartitionInfo/index.tsx:69]: replace it with Icon Component in /assets/components/Icon

File: app/assets/modules/ServiceManage/ConfigInfo/index.less [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+			background: #ecf0ff;
+		}
+
+		.meta {
+			background: #e2f8f7;
+		}
+
+		.ant-table-summary {
+			.ant-table-cell {
+				color: #4372ff;
```
Comment (hetao92) [inline app/assets/modules/ServiceManage/ConfigInfo/index.less:37]: we can put some common color into common.less and import variable，#4372ff has been named as '@blue' in common.less

File: app/assets/modules/ServiceManage/PartitionInfo/index.less [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+			background: #ecf0ff;
+		}
+
+		.meta {
+			background: #e2f8f7;
+		}
+
+		.ant-table-summary {
+			.ant-table-cell {
+				color: #4372ff;
```
Comment (hetao92) [inline app/assets/modules/ServiceManage/PartitionInfo/index.less:37]: use `@blue`

File: app/assets/modules/ServiceManage/ServiceInfo/index.tsx [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+      default:
+        break;
+    }
+  }
+
+  renderTooltip=text=>{
+    return <Tooltip placement="top" title={text} > 
+      <svg  aria-hidden="true">
+        <use xlinkHref={'#iconhelp'}></use>
+      </svg>
```
Comment (hetao92) [inline app/assets/modules/ServiceManage/ServiceInfo/index.tsx:74]: import <Icon />

File: app/assets/modules/Layout/routes.tsx [hunk 7]
```diff
@@ -31,22 +36,27 @@ export const MenuList = [{
...
   },{
     key: 'snapshot',
     title: intl.get('common.snapshot'),
     icon: '#iconnav-snapshot',
+    path: '/snapshot'
   },{
     key: 'longTermTask',
     title: intl.get('common.longTermTask'),
     icon: '#iconnav-timeConsuming',
+    path: '/time-consuming'
```
Comment (nianiaJR) [inline app/assets/modules/Layout/routes.tsx:59]: long-term-task, better to keep same with your component

Comment (nianiaJR) [inline app/assets/modules/Layout/routes.tsx:59]: long-term-task, better to keep same with your component

File: app/assets/modules/Layout/routes.tsx [hunk 8]
```diff
@@ -80,4 +90,29 @@ export const RoutesList = [
...
+    exact: true,
+  },
+  {
+    path: '/snapshot',
+    component: Snapshot,
+    exact: true,
+  },
+  {
+    path: '/time-consuming',
+    component: TimeConsuming,
```
Comment (nianiaJR) [inline app/assets/modules/Layout/routes.tsx:115]: TimeConsuming or LongTermTask?

File: app/assets/modules/ServiceManage/ConfigInfo/index.tsx [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+    switch (module) {
+      case 'Storage':
+        return <span className="module storage">{module}</span>
+      case 'Graph':
+        return <span className="module graph">{module}</span>
+      case 'Meta':
+        return <span className="module meta">{module}</span>
+      default:
+        break;
+    }
```
Comment (nianiaJR) [inline app/assets/modules/ServiceManage/ConfigInfo/index.tsx:59]: no need to do this, module.toLowerCase() make the uppercase to lowercase

File: app/assets/modules/ServiceManage/PartitionInfo/index.tsx [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+    switch (module) {
+      case 'Storage':
+        return <span className="module storage">{module}</span>
+      case 'Graph':
+        return <span className="module graph">{module}</span>
+      case 'Meta':
+        return <span className="module meta">{module}</span>
+      default:
+        break;
+    }
```
Comment (nianiaJR) [inline app/assets/modules/ServiceManage/PartitionInfo/index.tsx:63]: no need to use switch here, use `String.toLowerCase()`

File: app/assets/modules/ServiceManage/ServiceInfo/index.tsx [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+    switch (module) {
+      case 'Storage':
+        return <span className="module storage">{module}</span>
+      case 'Graph':
+        return <span className="module graph">{module}</span>
+      case 'Meta':
+        return <span className="module meta">{module}</span>
+      default:
+        break;
+    }
```
Comment (nianiaJR) [inline app/assets/modules/ServiceManage/ServiceInfo/index.tsx:68]: modify too

File: app/assets/modules/ServiceManage/TimeConsuming/index.tsx [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+        return <span className="running">{status}</span>
+      case 'QUEUE':
+        return <span className="queue">{status}</span>
+      case 'FAILED':
+        return <span className="failed">{status}</span>
+      case 'STOPPED':
+        return <span className="stopped">{status}</span>
+      default:
+        break;
+    }
```
Comment (nianiaJR) [inline app/assets/modules/ServiceManage/TimeConsuming/index.tsx:77]: same

**Review by nianiaJR — APPROVED** [review]
LGTM

**Pull request merged**